src/components/Articles/ArticleDetail.jsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { format } from 'date-fns';
+
+import { getArticleById } from '../../api/api';
+
+// Components
+import Spinner from '../Basic/Spinner';
+import ErrorMsg from '../Basic/ErrorMsg';
+
+const ArticleDetail = () => {
+  const { article_id } = useParams();
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+  const [article, setArticle] = useState(null);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getArticleById(article_id)
+      .then((article) => {
+        setArticle(article);
+        setError(null);
+        setIsLoading(false);
+      })
+      .catch(({ data: { error } }) => {
+        setError({ status: error.status, msg: error.message });
+        setIsLoading(false);
+      });
+  }, [article_id]);
+
+  let createdDate;
+
+  if (article) createdDate = format(new Date(article.created_at), 'dd/MM/yyyy');
```

Comment:
Not sure what this if statement is doing? Is there a time where this article would be undefined?